Rakefile

```diff
@@ -9,18 +9,39 @@ require "packaging/configuration"
 Packaging::Tasks.load_tasks(:exclude => ["tarball.rake"])
 
 require "yast/tasks"
-Yast::Tasks.submit_to(ENV.fetch("YAST_SUBMIT", "factory").to_sym)
+yast_submit = ENV.fetch("YAST_SUBMIT", "factory").to_sym
+Yast::Tasks.submit_to(yast_submit)
 
 Packaging.configuration do |conf|
   conf.package_name.sub!(/-.*/, "") # strip branch name
   conf.package_dir    = ".obsdir" # Makefile.ci puts it there
   conf.skip_license_check << /.*/
 
   # defined in Rakefile in https://github.com/openSUSE/packaging_rake_tasks
-  conf.obs_api = "https://api.opensuse.org/"
-  conf.obs_project = "filesystems:snapper"
-  conf.obs_target = "openSUSE_Factory"
-  conf.obs_sr_project = "filesystems:snapper"
+  if yast_submit == :sle12sp3
+    # IBS: Devel:YaST:Head/snapper
+    # see https://github.com/yast/yast-rake/blob/master/data/targets.yml
+    conf.obs_api = "https://api.suse.de/"
```

Comment:
This configuration is not the same defined in https://github.com/yast/yast-rake/blob/master/data/targets.yml#L30 ?